css/styles.css

```diff
@@ -182,14 +183,27 @@ Colors:
 .btn {
   display: flex;
   align-items: center;
+  justify-content: center;
+  cursor: pointer;
```

Comment:
`button` has is by default

Reply:
In Chrome, `cursor:pointer` is not used by default.